src/components/VLink.vue

```diff
@@ -12,7 +15,37 @@
       :size="4"
       rtl-flip
     />
-  </Component>
+  </NuxtLink>
+  <a
+    v-else-if="linkComponent === 'a'"
+    v-bind="$attrs"
+    :href="href"
+    target="_blank"
+    rel="noopener noreferrer"
+    :class="{ 'inline-flex flex-row items-center gap-2': showExternalIcon }"
+    v-on="$listeners"
+  >
+    <slot /><VIcon
+      v-if="showExternalIcon && !isInternal"
+      :icon-path="externalLinkIcon"
+      class="inline-block"
+      :size="4"
+      rtl-flip
+    />
+  </a>
+  <span
+    v-else
+    v-bind="$attrs"
+    :class="{ 'inline-flex flex-row items-center gap-2': showExternalIcon }"
+  >
+    <slot /><VIcon
+      v-if="showExternalIcon && !isInternal"
+      :icon-path="externalLinkIcon"
+      class="inline-block"
+      :size="4"
+      rtl-flip
+    />
+  </span>
```

Comment:
This isn't related to the changes in this PR, but wanted to call it out. This `span` fallback case worries me quite a bit :disappointed: What are the circumstances when we wouldn't have an `href` to pass to the link? Is it an asynchronous data issue or something like that?

Reply:
Yes, it's the asynchronous data issue, when we are trying to render VLink before the `href` was loaded, probably. And if I remember correctly, it happens on server rendering and is replaced with either `<a>` or `<NuxtLink>` on hydration.
We could (should?) probably audit the code and use `v-if` to not render `VLinks` before `href` is available.

Reply:
> We could (should?) probably audit the code and use v-if to not render VLinks before href is available.

I'd worry about pop-in at this point too though. We'd probably want that auditing to also focus on making sure page navigations work smoothly as data comes in.